packages/components-react/src/Checkbox.tsx

```diff
@@ -1,27 +1,11 @@
 import { CheckboxProps, Checkbox as UtrechtCheckbox } from '@utrecht/component-library-react';
-import clsx from 'clsx';
-import { ForwardedRef, forwardRef, PropsWithChildren, useId } from 'react';
+import { ForwardedRef, forwardRef, PropsWithChildren } from 'react';
 
 export { type CheckboxProps };
 
 export const Checkbox = forwardRef(
-  (
-    { id, className, name, value, invalid, ...restProps }: PropsWithChildren<CheckboxProps>,
-    ref: ForwardedRef<HTMLInputElement>,
-  ) => {
-    const checkboxId = id || useId();
-
-    return (
-      <UtrechtCheckbox
-        className={clsx('utrecht-checkbox', className)}
-        id={checkboxId}
-        invalid={invalid}
-        name={name}
-        ref={ref}
-        value={value}
-        {...restProps}
-      />
-    );
+  ({ ...Props }: PropsWithChildren<CheckboxProps>, ref: ForwardedRef<HTMLInputElement>) => {
+    return <UtrechtCheckbox ref={ref} {...Props} />;
   },
 );
 
```

Comment:
Volgens mij kan dit hele bestand gewoon 1 regel zijn:
```suggestion
export { Checkbox, type CheckboxProps } from '@utrecht/component-library-react;
```
De wrapper heeft geen meerwaarde.
Zie voor voorbeelden waarbij wrapper is weggelaten bijvoorbeeld:

- Radio.tsx
- Separator.tsx
- Figure.tsx